web/src/app/lists/PaginatedList.tsx

```diff
@@ -277,12 +279,9 @@ export function PaginatedList(props: PaginatedListProps): JSX.Element {
     const len = items.length
 
     // explicitly set props to load more, if loader function present
-    const loadProps: any = {}
-    if (loadMore) {
-      loadProps.hasMore = true
-      loadProps.next = loadMore
-    } else {
-      loadProps.hasMore = false
+    const loadProps = {
+      hasMore: Boolean(loadMore),
+      next: loadMore || (() => {}),
```

Comment:
you may want to double check that this does not have any unintended consequences, since the comment indicates that `next` was explicitly _not_ set (`undefined` as opposed to `() => {}`) if `loadMore` is false